spacewalk/certs-tools/spacewalk-certs-tools.changes.mc.enable-sl-micro-60

```diff
@@ -0,0 +1 @@
+- calculate bootstrap repo path for SL-Micro 6
```

Comment:
Bit nitpicky, but maybe change to `SL Micro 6.0` ?

Reply:
changed